torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -171,47 +171,56 @@ class SubclassCreationMeta:
     flat_tensor_start_idx: int
     # The number of tensors that live in this subclass wrapper
     arg_count: int
```

Comment:
after reading the code, some invariants that I think are worth explicitly mentioning in the comments:

- `arg_count` is inclusive of the arg_counts of any inner tensor subclasses: If I have a TwoTensor and both of its inner elements are TwoTensors, then the `arg_count` of the outer-most sublass will be 4